### DISCUSSION TITLE
far2l: смена цветового оформления

### DISCUSSION THREAD

**Discussion by Mischutka** [elfmz/far2l#1932] [Q&A] [answered]
Подскажите, пожалуйста, как поменять цвет шрифта и фона для каталогов и исполняемых файлов?

С помощью опции F9/Параметры /Цвета удалось поменять цвета лишь для обычных файлов, цвет фона и шрифта для для каталогов и исполняемых файлов остался без изменений.

![far2l_colors](https://github.com/elfmz/far2l/assets/93511177/49591ff3-896e-43a2-8d6e-15fc964f67b1)

**Comment by akruphi** [comment]
Кроме выполненных Вами настроек цветов самого far2l и панелей также для различных каталогов/файлов, удовлетворяющих признакам, используйте **Параметры->Раскраска файлов и группы сортировки**.

Итоговые раскраски по Shift-F9 записываются в `~/.config/far2l` или `$XDG_CONFIG_HOME/far2l` в файлы:
- `palette.ini` — расцветка панелей
- `settings/colors.ini` — раскраска файлов и групп сортировки

**Reply by Mischutka** [reply]
> Параметры->Раскраска файлов и группы сортировки.

Я с тамошними настройками цвета не разобрался. :-(

**Reply by akruphi** [reply, marked as answer]
Там для каждой группы по **F4** открываете диалог редактирования и внизу в **──── Цвета файлов**
![изображение](https://github.com/elfmz/far2l/assets/92621645/f2407d6c-718c-4cce-ab29-4b9bff01b6dd)
для 1,3,5,7 меняете цвета.
Цвет символа пометки в 2,4,6,8 обычно менять не требуется - far их делает автоматически (для этого там остаётся выбор черный на чёрном).

**Reply by Mischutka** [reply]
Спасибо большое, всё "дозакрасилось". :-)

**Comment by Mischutka** [comment]
Теперь возник вопрос, как "дозакрасить" редактор? Почему-то у строк остались стандартные цвета.

![far2l_colors_editor](https://github.com/elfmz/far2l/assets/93511177/4c7889ec-9dd3-467f-8b0e-8b098e67713f)

**Comment by spnethw** [comment]
> Теперь возник вопрос, как "дозакрасить" редактор? Почему-то у строк остались стандартные цвета.

Тут уже нужно поиграться с настройками плагина FarColorer.
Находясь в редакторе: F11 - FarColorer - Настройка.
Там пункты "Цветовой стиль".

**Reply by Mischutka** [reply]
Спасибо, теперь и редактор "дозакрашен".
